### DISCUSSION TITLE
SMBX 1.3 worlds

### DISCUSSION THREAD

**Discussion by mlloewen** [Wohlstand/TheXTech#55] [General]
I have been enjoying the world you have included in the build.  Where can we download other worlds that are comparable.  As most that I have tried are built with lua thus are only semi playable. I am on Linux. Thanks Keep up you good work.  Looking forward to the PGE being playable.

**Comment by Wohlstand** [comment]
In a history there are a TON of episodes where created, some of them I had to collect on my server, there are some I can give: http://wohlsoft.ru/docs/Collected/Episodes/
At my local store I have something that I will need to pack and publish on my archive. Also, https://www.supermariobrosx.org/forums/viewtopic.php?t=6606 (it's reworked version)
this one https://rpgmaker.net/games/4216/ I played first time when I met SMBX, however, it has some bugs I fixed on my end, I would to pack it as "re-pack" and store at my server for public. This also a thing https://rpgmaker.net/games/5438/ , btw, I met this in a moment when it was an incomplete demo, and it's literally FIRST episode which I played and I met SMBX in a first time, I found it accidentally while google Mario-universe pictures :smirk:

**Comment by Wohlstand** [comment]
One note: if you see some mega-episodes where are custom default set was given (I mean these episodes are shipping with full SMBX set which was been partially customized, Sarasaland Adventure and "Lowser's Conquest Beta" there are a big examples), to make it work, you need to patch it with taking of some default TheXTech's assets from my package:
- Copy `graphics/ui` folder from TheXTech's assets
- From [THIS](https://github.com/WohlSoft/SMBX-Integration) config pack, get "music.ini" and "sounds.ini" to match original MP3 filenames (in a package which I did, I used my Pure-quality sounds and musics I did re-ripped myself or replaced with SPC/NES chiptunes)
- copy "thextech" executable and "thexitech.ini" file from the default set into root of customized SMBX package
- use GIFs2PNG from PGE toolchain and convert all default stuff into PNG (no need to do this with episodes, they all should work):
```
GIFs2PNG -w "/path/to/customized/SMBX/root/graphics/"
```
(by default all GIFs will be kept untouched, however, if you will specify "-r" flag, all old GIFs will be removed)

and then, I hope, it will work.

**Comment by Wohlstand** [comment]
Some episodes are suggested by my friend Olhi:
* https://www.supermariobrosx.org/view/episodes/
* https://web.archive.org/web/20151114075430/http://www.supermariobrosx.org/episodes.html
* https://www.supermariobrosx.org/forums/viewtopic.php?f=36&t=17194
* https://www.supermariobrosx.org/forums/viewtopic.php?f=36&t=18006
> Also, two others that I can recommend are "Another Adventure" and "Super RMN World".
* https://rpgmaker.net/games/4676/

**Comment by Wohlstand** [comment]
The **2k15 Summer Takeover** is also pretty compatible: http://archive.wohlsoft.ru/SMBX%20Episodes/2k15%20Summer%20Takeover.html

I took it to store it into my archive. You need to get the "2k15 Summer Takeover: 1.fun.1" episode to play, it's compatible with XTech.

**Comment by 0lhi** [comment]
I have made a list of Episodes that I played:

https://wohlsoft.ru/pgewiki/User:Olhi#Episode_Ratings

Once you gained some familiarity with SMBX, it's also worth to check out Eclipsed's "_Beating Every SMBX Episode_":

https://www.supermariobrosx.org/forums/viewtopic.php?f=69&t=23666
